engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/CustomAction.scala

```diff
@@ -0,0 +1,51 @@
+package pl.touk.nussknacker.engine.api.deployment
+
+import pl.touk.nussknacker.engine.api.ProcessVersion
+
+import java.net.URI
+
+/*
+This is an experimental/wip version.
```

Comment:
This is experimental version, API will change in the future

Reply:
:+1: